src/modules/allowlist/pool.ts

```diff
@@ -13,6 +13,9 @@ export async function allowlistPool(chainId: number, poolId: string) {
 
   const poolDescription = await getPoolSymbolFromContract(poolId, provider);
 
+  // Ignore LPB pools as they do not require allowlisting
+  if (poolDescription.endsWith('_LBP')) return
```

Comment:
We cannot cannot get LBP type from `getPoolTypeFromContract` I think that checking the symbol is a reliable method.

Reply:
Sure, seems fine to me.